cookietemple/lint/template_linter.py

```diff
@@ -329,6 +330,17 @@ def _strip_ansi_codes(self, string, replace_with=""):
 
         return ansi_escape.sub(replace_with, string)
 
+    @staticmethod
+    def get_calling_class() -> object:
```

Comment:
Why is this method/information needed in the first place?
A stack inspection of this is just super ugly